### ISSUE TITLE
Feature request for the function neb

### ISSUE THREAD

**Issue by scott-5** [Jan8be/metadynminer.py#8]
Hi developer, as in R version metadynminer, the function neb is a very good feature, may I ask if this feature can be implemented in advanced python version metadynminer?

**Comment by Jan8be** [comment]
Yes, we can definitelly add the neb function in the future. I didn't know how many users would need it, but seeing that there is the demand, we will implement it. I will work on it as soon as I can.
